worlds/stardew_valley/__init__.py

```diff
@@ -218,12 +217,13 @@ def precollect_starting_season(self):
         if self.options.season_randomization == SeasonRandomization.option_randomized_not_winter:
             season_pool = [season for season in season_pool if season.name != "Winter"]
 
-        starting_season = self.create_starting_item(self.random.choice(season_pool))
+        item1 = self.random.choice(season_pool)
+        starting_season = self.create_item(item1)
```

Comment:
I don't love the name `item1` and I'm not sure why this was split apart, but it doesn't really matter anyway

Reply:
I'm not sure either lol